test/sharness/t0054-dag-car-import-export.sh

```diff
@@ -105,29 +113,60 @@ EOE
   mkfifo pipe_testnet
   mkfifo pipe_devnet
 
-  test_expect_success "fifo import" '
+  test_expect_success "basic fifo import" '
     (
-        cat ../t0054-dag-car-import-export-data/lotus_testnet_export_128_shuffled_nulroot.car > pipe_testnet &
-        cat ../t0054-dag-car-import-export-data/lotus_devnet_genesis_shuffled_nulroot.car > pipe_devnet &
+        timeout 30 cat ../t0054-dag-car-import-export-data/lotus_testnet_export_128_shuffled_nulroot.car > pipe_testnet &
+        timeout 30 cat ../t0054-dag-car-import-export-data/lotus_devnet_genesis_shuffled_nulroot.car > pipe_devnet &
 
         do_import 0 \
           pipe_testnet \
           pipe_devnet \
           ../t0054-dag-car-import-export-data/combined_naked_roots_genesis_and_128.car \
-        | sort > basic_fifo_import_actual
+        > basic_fifo_import_actual
         result=$?
 
         wait
         exit "$result"
     )
   '
 
-  test_expect_success "remove fifos" '
-    rm pipe_testnet pipe_devnet
+  test_expect_success "basic fifo-import output as expected" '
+    test_cmp_sorted basic_import_expected basic_fifo_import_actual
+  '
+
+  reset_blockstore 0
+  reset_blockstore 1
+
+  test_expect_success "sequenced fifo import" '
+    (
+        # Part 2 is smaller, goes in first and blocks waiting on ipfs
+        # What is tested here is that by the time import 1 + roots finishes and the
+        # pipes are deleted, the preexisting open file handles will carry through,
+        # even though pipe_devnet is no longer on the filesystem
+        timeout 30 cat ../t0054-dag-car-import-export-data/lotus_devnet_genesis_shuffled_nulroot.car > pipe_devnet &
```

Comment:
I would remove all these timeouts or set them to several minutes. CI can be _very_ slow at times.